Document/0x06i-Testing-Code-Quality-and-Build-Settings.md

```diff
@@ -320,7 +320,51 @@ The dynamic analysis of this section comprises of two parts: the actual license
 
 It need to be validated whether the copyrights of the licenses have been adhered to. This often means that the application should have an `about` or `EULA` section in which the copy-right statements are noted as required by the license of the third party library.
 
-When no source-code is available for library analysis, you can find some of the frameworks being used with otool and MobSF.
+##### Listing application libraries
+
+When no source-code is available for libraries analysis, you can find some of the frameworks being used with [objection](https://github.com/sensepost/objection), [MobSF](https://github.com/MobSF/Mobile-Security-Framework-MobSF) or otool.
+
+###### objection
+
+Objection contains bundles module to work with iOS Bundles. This module has two commands `list_bundles` and `list_frameworks`.
+
+`list_bundles` command lists all of the application’s non framework bundles. The output of both commands contains executable name, bundle id, version of the library and path to the library.
+
+```bash
+...itudehacks.DVIAswiftv2.develop on (iPhone: 13.2.3) [usb] # ios bundles list_bundles
+Executable    Bundle                                       Version  Path
+------------  -----------------------------------------  ---------  -------------------------------------------
+DVIA-v2       com.highaltitudehacks.DVIAswiftv2.develop          2  ...-1F0C-4DB1-8C39-04ACBFFEE7C8/DVIA-v2.app
+CoreGlyphs    com.apple.CoreGlyphs                               1  ...m/Library/CoreServices/CoreGlyphs.bundle
+```
+
+`list_frameworks` command lists all of the application’s bundles that represent frameworks.
+
```

Comment:
Version might not be available since those frameworks are being provided by Apple, Other frameworks have versions.
But I can also spot apple bundle with the version, so it may also be objection bug.
`RawCamera     com.apple.RawCamera.bundle                 9.00.1     ...em/Library/CoreServices/RawCamera.bundle`